opendatahub-io/opendatahub-operator
# Add dashboard crds using manifests

Fixes #531 
## Description
This PR  introduces following changes:
- Deploy Dashboard CRDs only when odh-dashboard component is enabled
- Remove Dashboard CRDs from bundle
- Remove CRDs from validation test in e2e framework

## How Has This Been Tested?
1. Deploy test catalogsource
```
quay.io/vhire/opendatahub-operator-catalog:v2.1.10
```

2. Verify Dashboard CRDs are not installed as part of bundle
![Screenshot 2023-09-13 at 3 49 33 PM](https://github.com/opendatahub-io/opendatahub-operator/assets/17230536/fd9159bf-0dc6-42ef-8cba-ff7377f12e34)

3. Verify CRDs are deployed when dasboard is enabled
## Merge criteria:
